### PR TITLE
add git cheat sheet to docs

### PR DESCRIPTION
**Pull request by sophiad96**
## Description & motivation
This PR adds a git cheat sheet page (actually called The IDE Git Button) that provides users with guidance on what the different options that appear on the git button in the IDE will do.

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [X] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [X] The page has been added to `website/sidebars.js`
- [X] The new page has a unique filename

### CODE REVIEW

**Comment by sophiad96** [comment]
The images in this show as broken. Couldn't figure out why.

File: website/docs/docs/dbt-cloud/cloud-ide/the-ide-git-button.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: "The IDE Git Button"
+---
+
+The git button in the IDE allows you to apply the concept of [version control](dbt-cloud/cloud-ide/the-dbt-ide#version-control) to your project. This page provides descriptions of each git button command and what they do.
+
+**Initialize your project**: This is done when first setting up your project. Initializing a project will create all required directories and files within an empty repository. Note: This option will not display if your repo isn't completely empty (i.e. includes a README file).
+
+<p align="center">
+<Lightbox src="/img/initialize.png" />
```
Comment (clrcrl) [inline website/docs/docs/dbt-cloud/cloud-ide/the-ide-git-button.md]: re: the images are broken

You'll have to add the images to the repo as well — I think I go over that in the Loom vid in the README :)

For example, if I have an image reference like this:
```
<Lightbox src="/img/create-bigquery-project.png" title="Create a new GCP project" />
```
It picks up the image here: https://github.com/fishtown-analytics/docs.getdbt.com/tree/current/website/static/img

**Review by clrcrl — CHANGES_REQUESTED** [review]

File: website/docs/docs/dbt-cloud/cloud-ide/the-ide-git-button.md (hunk 1 above)
Comment (sophiad96) [reply website/docs/docs/dbt-cloud/cloud-ide/the-ide-git-button.md]: ah tysm!

**Comment by drewbanin** [comment]
removing myself from review, adding @saouderkirk

**Review by annafil — APPROVED** [review]
LGTM! 🎉 

Non-blocking suggestion: I would love to see these docs point to a really good Git reference -- e.g. "Not sure what these terms mean? Check out X for more details" 

I personally like [this one](https://rogerdudler.github.io/git-guide/) but you probably have your own favorite :)

**Pull request merged**